### PR TITLE
feat: create security release command

### PR DESCRIPTION
**Pull request by RafaelGSS**
This new command should manage all the states of a security release. This PR includes the `CREATE` and should be followed by other PRs to manage the next steps, such as requesting CVEs, creating issues, sending emails and so on.

Example this is action: https://github.com/nodejs-private/node-private/issues/443

### CODE REVIEW

**Comment by RafaelGSS** [comment]
ping @nodejs/security-release

**Comment by RafaelGSS** [comment]
ping @nodejs/releasers

image

File: lib/prepare_security.js [hunk 1]
```diff
@@ -0,0 +1,207 @@
+import nv from '@pkgjs/nv';
+import auth from './auth.js';
+import Request from './request.js';
+
+const TEMPLATE = `
+## Planning
+
+* [X] Open an [issue](https://github.com/nodejs-private/node-private) titled
```
Comment (aduh95) [inline lib/prepare_security.js:8]: It seems the template is static, would it make more sense as a separate `.md` file?

File: lib/prepare_security.js [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+      const content = await issue.buildIssue(cli);
+      const { repository: { id } } = await req.gql('RepositoryId', {
+        owner: 'nodejs-private',
+        repo: 'node-private'
+      });
+      const data = await req.gql('CreateIssue', {
+        repoId: id,
+        title: 'Next Security Release',
+        body: content
+      });
```
Comment (aduh95) [inline lib/prepare_security.js:132]: This should be one request instead of two, the GraphQL API is supposed to let us send fewer requests to GitHub, not more 😅

Comment (aduh95) [reply lib/prepare_security.js:132]: @RafaelGSS do you need help with the above?

**Comment by RafaelGSS** [comment]
Done @aduh95. Could you please re-review?

File: docs/git-node.md [hunk 3]
```diff
@@ -330,7 +330,7 @@ ncu-config set waitTimeMultiApproval 48
 
 ## `git node v8`
 
-Update or patch the V8 engine.  
+Update or patch the V8 engine.
```
Comment (aduh95) [inline docs/git-node.md:333]: Was this change on purpose? IIUC those spaces are here to tell the Markdown interpreter there should be a line break.

File: lib/github/templates/next-security-release.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
+## Planning
+
+* [X] Open an [issue](https://github.com/nodejs-private/node-private) titled
+  \`Next Security Release\`, and put this checklist in the description.
```
Comment (aduh95) [inline lib/github/templates/next-security-release.md:4]: I assume this is a copy/pasting error, right?
```suggestion
  `Next Security Release`, and put this checklist in the description.
```

File: lib/github/templates/next-security-release.md [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+      only create CVEs for vulnerabilities in Node.js itself. This is to avoid
+      having duplicate CVEs for the same vulnerability.
+  * Described in the pre/post announcements
+
+* [ ] Pre-release announcement to nodejs.org blog: TBD
+  (Re-PR the pre-approved branch from nodejs-private/nodejs.org-private to
+  nodejs/nodejs.org)
+
+* [ ] Pre-release announcement [email](https://groups.google.com/forum/#!forum/nodejs-sec): TBD
+  * Subject: \`Node.js security updates for all active release lines, Month Year\`
```
Comment (aduh95) [inline lib/github/templates/next-security-release.md:43]: Same here

File: lib/github/templates/next-security-release.md [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+
+* [ ] CC \`oss-security@lists.openwall.com\` on pre-release
+  * [ ] Forward the email you receive to \`oss-security@lists.openwall.com\`.
```
Comment (aduh95) [inline lib/github/templates/next-security-release.md:46]: Same here

File: lib/github/templates/next-security-release.md [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+    * (Check that the "Version Fixed" field in the CVE is correct, and provide
+      links to the release blogs in the "Public Reference" section)
+
+* [ ] PR machine-readable JSON descriptions of the vulnerabilities to the
+  [core](https://github.com/nodejs/security-wg/tree/HEAD/vuln/core)
+  vulnerability DB. https://github.com/nodejs/security-wg/pull/1029
+  * For each vulnerability add a \`#.json\` file, one can copy an existing
+    [json](https://github.com/nodejs/security-wg/blob/0d82062d917cb9ddab88f910559469b2b13812bf/vuln/core/78.json)
+    file, and increment the latest created file number and use that as the name
+    of the new file to be added. For example, \`79.json\`.
```
Comment (aduh95) [inline lib/github/templates/next-security-release.md:89]: Same here

File: lib/prepare_security.js [hunk 8]
```diff
@@ -0,0 +1,119 @@
...
+  }
+
+  getSecurityIssueTemplate() {
+    return fs.readFileSync(
+      path.resolve(
+        'lib',
+        'github',
+        'templates',
+        'next-security-release.md'
+      )
```
Comment (aduh95) [inline lib/prepare_security.js:54]: Let's not make this dependent on the cwd
```suggestion
      new URL(
        './github/templates/next-security-release.md',
        import.meta.url
      )
```

File: lib/prepare_security.js [hunk 9]
```diff
@@ -0,0 +1,119 @@
...
+    ).toString();
```
Comment (aduh95) [inline lib/prepare_security.js:55]:
```suggestion
    , 'utf-8');
```

File: docs/git-node.md (hunk 3 above)
Comment (RafaelGSS) [reply docs/git-node.md:333]: Possibly it was caused by `npm run lint-fix`

Comment (aduh95) [reply docs/git-node.md:333]: Let's add them back and see if the linter complains
```suggestion
Update or patch the V8 engine.  
```

File: docs/git-node.md [hunk 10]
```diff
@@ -367,7 +367,7 @@ Options:
 ### `git node v8 minor`
 
 Compare current V8 version with latest upstream of the same major. Applies a
-patch if necessary.  
+patch if necessary.
```
Comment (aduh95) [inline docs/git-node.md:370]:
```suggestion
patch if necessary.  
```

**Review by aduh95 — APPROVED** [review]
